lib/govuk_message_queue_consumer/rabbitmq_config.rb

```diff
@@ -10,7 +10,24 @@ def self.from_environment(env)
         user: fetch(env, "RABBITMQ_USER"),
         pass: fetch(env, "RABBITMQ_PASSWORD"),
         recover_from_connection_close: true,
-      }
+      }.merge(tls_config(env))
+    end
+
+    def self.tls_config(env)
+      # TLS config is optional - local RabbitMQs won't use TLS,
+      # but Amazon MQ enforces it. So we default to non-TLS.
+      if env.fetch("RABBITMQ_TLS", false)
+        {
+          tls: true,
+          tls_cert: env.fetch("RABBITMQ_TLS_CERT", nil),
+          tls_key: env.fetch("RABBITMQ_TLS_KEY", nil),
+          tls_protocol: env.fetch("RABBITMQ_TLS_VERSION", :TLSv1_2).to_sym,
+          tls_ca_certificates: env.fetch("RABBITMQ_TLS_CA_CERTIFICATES", "").split(","),
+          verify_peer: env.fetch("RABBITMQ_VERIFY_PEER", false),
```

Comment:
Out of curiosity, why do we want this off?

Reply:
I set to off by default, so that you can develop & test locally against an AmazonMQ instance without needing to worry about the certificates.